README.md

```diff
@@ -421,6 +431,7 @@ Python | 3.6 or higher | No | *N/A* | *Pre-installed or Installed by user* | [PS
 [NVIDIA CUDA SDK](https://developer.nvidia.com/cublas) | 10.2 | No | *N/A* | *Installed by user* |[End User License Agreement](https://docs.nvidia.com/cuda/eula/index.html)
 [AMD rocBLAS](https://rocblas.readthedocs.io/en/rocm-4.5.2/) | 4.5 | No | *N/A* | *Installed by user* |[AMD License](https://github.com/ROCmSoftwarePlatform/rocBLAS/blob/develop/LICENSE.md)
 [AMD rocRAND](https://github.com/ROCmSoftwarePlatform/rocRAND) | 5.1.0 | No | *N/A* | *Installed by user* |[AMD License](https://github.com/ROCmSoftwarePlatform/rocRAND/blob/develop/LICENSE.txt)
+[AMD rocSOLVER](https://github.com/ROCmSoftwarePlatform/rocSOLVER) | 5.0.0 | No | *N/A* | *Installed by user* |[AMD License](https://github.com/ROCmSoftwarePlatform/rocRAND/blob/develop/LICENSE.txt)
```

Comment:
Is there a reason why the 5.0.0 version of rocSOLVER is used? It doesn't match either the rocBLAS version (4.5.2) nor the rocRAND version (5.1.0, first version with a needed bug fix). If possible, it would be good if the rocSOLVER version can align to the rocBLAS version as there may be a need to use both domains in an app.
Also, the License URL looks like it's pointing to rocRAND's license.

Reply:
Thanks for pointing the wrong license link with rocSOLVER. This will be updated in the next commit.

We currently don't have a machine with ROCm installation other than 5.0. We'll address this concern soon and until then we'll treat this as an open issue.

Reply:
Thanks for the clarification! Do the current validation results attached to the description contain all of the changes made during the review (in other words, were the validation results done using the latest version of this branch)? If not, please do one final round of validation testing to ensure that the latest changes did not introduce any new issues. Thanks!

Reply:
Yes, the test case logs reflect the validation of the latest code base